main/acle.md

```diff
@@ -8144,8 +8144,8 @@ boundaries. This can be useful in the following example situations:
     switches should be avoided for performance reasons.
 
 *   A function provides a public API that is specific to SME.
-    Again, callers to such functions would want to avoid the
-    overhead of switching modes at function call boundaries.
+    Again, callers to such functions should avoid the overhead
```

Comment:
I don't think this change is correct.  The text isn't saying what callers *should* do (what the ACLE requires them to do).  It's explaining what callers would want to happen (for performance reasons).

Reply:
Thanks for pointing that out @rsandifo-arm . I will revert this particular change.